nav2_bt_navigator/src/navigators/navigate_through_poses.cpp

```diff
@@ -164,8 +169,21 @@ NavigateThroughPosesNavigator::onLoop()
 
     feedback_msg->distance_remaining = distance_remaining;
     feedback_msg->estimated_time_remaining = estimated_time_remaining;
-  } catch (...) {
```

Comment:
Thus, I think all of this can/should be removed

Reply:
exceptions removed in favor of logic.